edx-platform/xblocks/xblocks_i18n.py

```diff
@@ -0,0 +1,48 @@
+"""
+Push and pull the latest sources of XBlocks to Transifex in batch.
+"""
+from __future__ import print_function, unicode_literals
+
+from yaml import safe_load
+from subprocess import check_call
+from shutil import rmtree
+from os import system, mkdir
+from os import path
+import time
+
+XBLOCKS_DIR = path.dirname(__file__)
+
+def xblock_configs():
+    with open(path.join(XBLOCKS_DIR, 'config.yaml'), 'r') as config_file:
+        config = safe_load(config_file)
+
+    for xblock in config['xblocks']:
+        yield xblock
+
+
+def pull_translations():
+    for config in xblock_configs():
+        repos_dir = path.join(XBLOCKS_DIR, 'repos')
+        repo_dir = path.join(repos_dir, config['name'])
+
+        branch = 'i18n-bot/{time}'.format(time=time.strftime('%Y-%m-%d-%H%M%S'))
+
+        check_call(['git', 'clone', config['upstream_repo'], config['name']], cwd=repos_dir)
+        check_call(['git', 'remote', 'add', 'local', config['local_repo']], cwd=repo_dir)
+        check_call(['git', 'checkout', '-b', branch, 'master'], cwd=repo_dir)
+
+        # check_call(config['requirements_script'], shell=True, cwd=repo_dir)
+        check_call(config['pull_script'], shell=True, cwd=repo_dir)
+
+        # TODO: Change to something owned by the Open edX team
+        check_call(['git', 'config', 'user.name', 'Open edX i18n Bot'], cwd=repo_dir)
+        check_call(['git', 'config', 'user.email', 'omar+openedx-i18n-bot@appsembler.com'],
```

Comment:
I don't think we need to change to a real url. I'd be fine with `bot@openedxi18n.org`.